structured/keys.go

```diff
@@ -35,10 +35,34 @@ func MakeNameMetadataKey(parentID ID, name string) proto.Key {
 	return k
 }
 
-// MakeDescMetadataKey returns the key for the table in namespaceID.
+// MakeDescMetadataKey returns the key for the descriptor.
 func MakeDescMetadataKey(descID ID) proto.Key {
 	k := make([]byte, 0, len(keys.DescMetadataPrefix)+encoding.MaxUvarintSize)
 	k = append(k, keys.DescMetadataPrefix...)
 	k = encoding.EncodeUvarint(k, uint64(descID))
 	return k
 }
+
+// EncodeColumnKey does stuff
```

Comment:
I'd prefer to see `EncodeColumnKey TODO: undocumented`. Or document this. Ditto for the other `Encode` methods below.